.build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Get path from where we are 
+LOCATION="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
I wish we didn't have to make this bash-specific but I don't know of a good, sh-compatible way to get the directory that the currently executing script is located in.